source/includes/api.html.md.erb

```diff
@@ -1243,7 +1274,7 @@ Para realizar esta operación es obligatorio utilizar tu <strong><a href="#confi
   
 </ul>
 
-Entrega una lista de industrias filtradas por nombre.
+Entrega una lista de industrias filtradas por nombre del idioma del usuario.
```

Comment:
are you sure they are filtered by language?

Reply:
In a Cristian commit on clientela changes this line https://github.com/admetricks/clientela/commit/bad2a127d27aa3f1a9040f1d2bbf42c19c602f0b

Reply:
![image](https://user-images.githubusercontent.com/5798343/214922166-9a7155ca-b4be-4254-b9b1-b55ae7fc8b27.png)

Reply:
@vitorohe

Reply:
so, industries come in the language of the user?
```
Entrega una lista de industrias filtradas por nombre en el idioma del usuario
```

Reply:
I think, yes 
![image](https://user-images.githubusercontent.com/5798343/215090059-68dc4bb9-db96-4051-9ca1-61488359b74e.png)